Project.toml

```diff
@@ -1,6 +1,6 @@
 name = "Singular"
 uuid = "bcd08a7b-43d2-5ff7-b6d4-c458787f915c"
-version = "0.2.2"
+version = "0.2.3"
```

Comment:
Perhaps this should be 0.3.0 as there is a binary incompatibility due to the CxxWrap switch? Honestly, though, I don't think it matters in practice for our purposes *shrug*